AnimateQuotes.js

```diff
@@ -82,7 +97,7 @@ export default function AnimateQuotes(props) {
       height: '100%',
       width: '100%',
     }}>
-      {props.children}
+      <Quotes quote={{ quote }} setQuote={{ setQuote }} />
```

Comment:
Nesting child component in AnimateQuotes return statement for simplification of passing props instead of using props.children which became unnecessary due to child component not requiring dynamism.

Reply:
Update this API so you don't pass objects
```suggestion
      <Quotes quote={ quote } setQuote={ setQuote } />
```